### PR TITLE
Add more smoke tests

### PR DESCRIPTION
**Pull request by Wawrzyn321**
**Description**

Tests proposed in this pull request:
* Luigi:
  * Luigi navigation (already present)
* Add-ons:
  * Cluster Addons view
* Core
  * Applications view (already present)
  * Deployments view
* Core-ui
  * Namespaces view (already present)
* Functions:
  * Functions view
* Logging:
  * Logs view
* Service-catalog-ui
  * Catalog view (already present)
  * Service Brokers view
  * Instances view
* Content:
  * Docs view

**Related issue(s)**
Continues [6212](https://github.com/kyma-project/kyma/issues/6212)
[Kyma](https://github.com/kyma-project/kyma/pull/8116)

### CODE REVIEW

File: tests/tests/tests.js [hunk 1]
```diff
@@ -52,12 +77,82 @@ testIf(
...
+  await retry(t, 3, t => {
+    return t
+      .expect(
+        Selector('.fd-action-bar__title').withText(/Cluster Addons/i).exists,
+      )
+      .ok();
+  });
+});
+
+test('Functions view is rendered', async t => {
```
Comment (dariadomagala) [inline tests/tests/tests.js:107]: I'm not 100% sure, but I think that functions are also a pluggable (although turned on by default)

File: tests/tests/tests.js [hunk 2]
```diff
@@ -36,6 +36,31 @@ test('Namespaces view is rendered', async t => {
...
+      Selector('iframe', { timeout: 20000 }).filterVisible(),
+    );
+    return t
+      .expect(Selector('.fd-action-bar__title').withText('Deployments').exists)
+      .ok();
+  });
+});
+
 testIf(
   !toBoolean(config.apiPackagesEnabled),
```
[the remark this answers is not in the document]
Comment (Wawrzyn321) [reply tests/tests/tests.js:65]: Good point!

[the remark this answers is not in the document]
Comment (Wawrzyn321) [reply tests/tests/tests.js]: I've changed it also in Kyma repo PR.

File: tests/tests/tests.js (hunk 1 above)
Comment (Wawrzyn321) [reply tests/tests/tests.js]: Fixed in 0ab6734.

File: tests/tests/tests.js [hunk 3]
```diff
@@ -52,12 +77,82 @@ testIf(
...
+  //THEN
+  await retry(t, 3, findActiveFrame);
+  await retry(t, 3, t => {
+    return t
+      .expect(Selector('.fd-panel__title').withText(/Functions/).exists)
+      .ok();
+  });
+});
+
+test('Logs view is rendered', async t => {
```
[the remark this answers is not in the document]
Comment (Wawrzyn321) [reply tests/tests/tests.js]: Fixed in 0ab6734.

File: tests/tests/tests.js [hunk 4]
```diff
@@ -36,6 +36,31 @@ test('Namespaces view is rendered', async t => {
   });
 });
 
+test('Deployments view is rendered', async t => {
+  //GIVEN
+  await t.useRole(adminUser);
```
[the remark this answers is not in the document]
Comment (Wawrzyn321) [reply tests/tests/tests.js:41]: I'll give it a shot.

File: tests/tests/tests.js [hunk 5]
```diff
@@ -36,34 +34,126 @@ test('Namespaces view is rendered', async t => {
...
+    return t
+      .click(Selector('.fd-panel__title').withText('default'))
+      .switchToMainWindow()
+      .click(deploymentsLink);
+  });
+
+  //THEN
+  await retry(t, 3, async t => {
+    await t.switchToIframe(
+      Selector('iframe', { timeout: 20000 }).filterVisible(),
```
Comment (dariadomagala) [inline tests/tests/tests.js:53]: Why not `findActiveFrame` as it is in other places?

Comment (Wawrzyn321) [reply tests/tests/tests.js:53]: Using `findActiveFrame` fails with `The element that matches the specified selector is not visible.`

Comment (dariadomagala) [reply tests/tests/tests.js:53]: Even with 3 retries? That's weird

**Comment by dariadomagala** [comment]
./test all

**Pull request merged**